src/business-logic/device-factories/device-connection-factory.ts

```diff
@@ -0,0 +1,26 @@
+import { DeviceType } from '../../model/enums/device-type'
+import { Device } from '../../model/entities/device'
+import { DeviceConnection } from '../services/interfaces/device-connection'
+import { INewsGatewayDeviceConnection } from '../services/inews-gateway-connection-implementation'
+import { DeviceConnectionFactory } from '../services/interfaces/device-connection-factory'
+import { Logger } from '@tv2media/logger/*'
+import { LoggerFacade } from '../../logger/logger-facade'
+
+export class DeviceConnectionFactoryImplementation implements DeviceConnectionFactory {
+  constructor(private readonly logger: Logger){
+  }
+  
+  public createDeviceConnection(device: Device): DeviceConnection | undefined {
+    return this.mapDeviceToDeviceConnection(device)
+  }
+
+  public mapDeviceToDeviceConnection(device: Device): DeviceConnection | undefined {
```

Comment:
This method is only use in this class so it should be private

Reply:
Private now.